src/Gridify/Syntax/OperatorManager.cs

```diff
@@ -8,7 +8,7 @@ namespace Gridify.Syntax;
 
 public class OperatorManager
 {
-   private ConcurrentDictionary<string, IGridifyOperator>? _operators;
+   private readonly ConcurrentDictionary<string, IGridifyOperator> _operators = new();
    public IEnumerable<IGridifyOperator> Operators => _operators?.Values ?? Enumerable.Empty<IGridifyOperator>();
```

Comment:
`_operators` can't be null anymore

Reply:
I guess, you can now remove all the `_operators?` calls, which is great.

Reply:
Which is a bug fix as well. Since `OperatorManager` is a Singleton, the dictionary should be instatiaded only once. Previously the there was a risk of being instantiated multiple times, resulting in a possible loss of operator registering.